profiler/src/main/java/com/navercorp/pinpoint/profiler/instrument/PlainClassLoaderHandler.java

```diff
@@ -102,7 +102,9 @@ private ClassPool createClassPool(ClassLoader classLoader) throws NotFoundExcept
     private Class<?> injectClass0(ClassPool pool, ClassLoader classLoader, String className, ClassLoadingChecker classLoadingChecker) throws NotFoundException, IOException, CannotCompileException, IllegalArgumentException, IllegalAccessException, InvocationTargetException {
         Class<?> c = null;
         try {
-            c = classLoader.loadClass(className);
+            // Using different approach to load classes via classloader
+            // c = classLoader.loadClass(className);
+            c = Class.forName(className, false, classLoader);
```

Comment:
Hi @emeroad 

Classloader can load classes via both the ways:
1) classLoader.loadClass 
2) Class.forName 

With classLoader.loadClass I was facing linkage error for example - "Caused by: java.lang.LinkageError: loader (instance of  org/jboss/modules/ModuleClassLoader): attempted  duplicate class definition for name: "com/navercorp/pinpoint/plugin/jdbc/mysql/MySqlConstants", while agent was instrumenting class in jboss environment. 

I looked at the alternative ways to load classes in multithreaded environment as the issue of linkage error has been reported by many (not specifically with jboss) with classloader.loadclass.

I tried to load class with class.forname it worked for me without any issues for jboss. To be absolutely sure that it won't break anything, I tested it with Tomcat environment as well and it worked fine.

Let me know your thoughts.

Thanks,
Suraj